components/navbar/Navbar.js

```diff
@@ -3,10 +3,18 @@ import AppTitleAndLogo from "./AppTitleAndLogo";
 import UserBubble from "./UserBubble";
 import DesktopMenu from "./DesktopMenu";
 import MobileMenu from "./MobileMenu";
-
-const pages = [{ text: "Home", url: "/" }];
+import useFetch from "../../hooks/use-fetch";
+import Loader from "../common/Loader";
 
 function Navbar() {
+  const { data, error } = useFetch(`/api/curriculums`);
```

Comment:
I think this could be a context if it's being used everywhere. We could then find the selected curriculum and put its title on every page as the first element of a breadcrumb for instance